examples/state-usage/server.js

```diff
@@ -0,0 +1,74 @@
+const express = require('express');
+const passport = require('passport');
+const { Strategy } = require('@superfaceai/passport-twitter-oauth2');
+const session = require('express-session');
+require('dotenv').config();
+
+passport.serializeUser(function (user, done) {
+  done(null, user);
+});
+passport.deserializeUser(function (obj, done) {
+  done(null, obj);
+});
+
+// Use the Twitter OAuth2 strategy within Passport
+passport.use(
+  new Strategy(
+    {
+      clientID: process.env.TWITTER_CLIENT_ID,
+      clientSecret: process.env.TWITTER_CLIENT_SECRET,
+      clientType: 'confidential',
+      callbackURL: `${process.env.BASE_URL}/auth/twitter/callback`,
+    },
+    (accessToken, refreshToken, profile, done) => {
+      console.log('Success!', { accessToken, refreshToken });
+      return done(null, profile);
+    }
+  )
+);
+
+const app = express();
+
+app.use(passport.initialize());
+app.use(
+  session({ secret: 'keyboard cat', resave: false, saveUninitialized: true })
+);
+
+app.get(
```

Comment:
suggestion: I would state here explicitly, that the state has to be object

```
app.get(
  '/auth/twitter',
  (req, res, next) => {
    const stateObject = {
      key: req.query.state
    };

    passport.authenticate('twitter', {
      scope: ['tweet.read', 'users.read', 'offline.access'],
      state: stateObject // Passing the state as an object is required by the Passport strategy
    })(req, res, next);
  }
);
```

Reply:
yes, this makes perfect sense. I have updated the PR with suggested changes